### PR TITLE
Add GitHub actions

### PR DESCRIPTION
**Pull request by thabarbados**
Tune build config, update size-limit, add Github actions for tests and check commit name.

### CODE REVIEW

File: .github/workflows/checks.yml [hunk 1]
```diff
@@ -0,0 +1,28 @@
+name: Checks
+on:
+  pull_request:
+    branches:
+      - main
```
Comment (dangreen) [inline .github/workflows/checks.yml:5]: master

File: rollup.config.js [hunk 2]
```diff
@@ -58,6 +61,9 @@ export default [
     output: {
       format: 'umd',
```
Comment (dangreen) [inline rollup.config.js:62]: maybe drop this?

**Review by dangreen — CHANGES_REQUESTED** [review]

File: .github/workflows/checks.yml (hunk 1 above)
Comment (thabarbados) [reply .github/workflows/checks.yml]: changed

File: rollup.config.js (hunk 2 above)
Comment (thabarbados) [reply rollup.config.js]: dropped

**Review by dangreen — APPROVED** [review]

**Pull request merged**